cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1,1525 +1,1554 @@
-###################################################################################################################
```

Comment:
Why does git see that this entire file changed? Can you check the whitespace for unexpected changes?

Reply:
ahh because I committed the file with CLRF. :)

Reply:
fixed